ldregistry/templates/about/new_register.vm

```diff
@@ -0,0 +1,41 @@
+
+#parse("header.vm")
+#set($nav="about")
+#parse("navbar.vm")
+
+    <div class="jumbotron">
+      <div class="container">
+        <h1>How to provide a new register of content</h1>
+        <a href="http://www.wmo.int/pages/index_en.html">
+            <img class="float-right"  src="$uiroot/img/wmo-logo-official-blue-on-white-trunc.png" alt="World Meteorological Organization" name="logoWMO" width="169" height="144" border="5"></img>
+        </a>
+      </div>
+    </div>
+
+    <div class="container">
+      <div class="row">
+ <p> <a href="$uiroot/about/new_register">How to provide a new register of content.</a> </p>
+      <p> → Add explanation that WMO will approve it (or not)
```

Comment:
The WMO have numerous expert teams and task teams.  Relevant teams and the WMO secretariat are responsible for approving the exact content to be published within the WMO name space.